Felienne/hedy
# Fix docker build file

Dependency `ruamanel` depends on `gcc` to be present (at least on ARM64 a.k.a. Apple Silicon) in the Dockerfile, which it was not. Adding this to the build fixes it. This will make the docker image a bit larger though. 

**Description**

Fixed docker build, perhaps on all platforms, perhaps just on ARM64.

**Fixes _issue or discussion number_**

👼  I was hoping that wasn't necessary for a build fix

**How to test**

Run `docker build -t hedy .` as described in `CONTRIBUTING.md`

Error before the fix

```
#7 4.817 Collecting ruamel.yaml.clib>=0.1.2
#7 4.831   Downloading ruamel.yaml.clib-0.2.6.tar.gz (180 kB)
#7 4.836      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 180.7/180.7 KB 52.6 MB/s eta 0:00:00
#7 4.850   Preparing metadata (setup.py): started
#7 4.945   Preparing metadata (setup.py): finished with status 'error'
#7 4.949   error: subprocess-exited-with-error
#7 4.949   
#7 4.949   × python setup.py egg_info did not run successfully.
#7 4.949   │ exit code: 1
#7 4.949   ╰─> [3 lines of output]
#7 4.949       sys.argv ['/tmp/pip-install-0s6gha98/ruamel-yaml-clib_2bdbbb191e1c4922beb62d1c447e3657/setup.py', 'egg_info', '--egg-base', '/tmp/pip-pip-egg-info-flipexk8']
#7 4.949       test compiling /tmp/tmp_ruamel_23lbu7qn/test_ruamel_yaml.c -> test_ruamel_yaml compile error: /tmp/tmp_ruamel_23lbu7qn/test_ruamel_yaml.c
#7 4.949       Exception: command 'gcc' failed: No such file or directory
#7 4.949       [end of output]
#7 4.949   
#7 4.949   note: This error originates from a subprocess, and is likely not a problem with pip.
#7 4.950 error: metadata-generation-failed
#7 4.950 
#7 4.950 × Encountered error while generating package metadata.
#7 4.950 ╰─> See above for output.
#7 4.950 
#7 4.950 note: This is an issue with the package mentioned above, not pip.
#7 4.950 hint: See above for details.
#7 5.039 WARNING: You are using pip version 22.0.4; however, version 22.2.2 is available.
#7 5.039 You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
```

After the fix *it just works* ™️
